# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-03-31 11:33+0800\n"
"PO-Revision-Date: 2021-03-31 11:39+0800\n"
"Last-Translator: afa005 <afa@openpilot.net>\n"
"Language-Team: \n"
"Language: zh_TW\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 2.4.2\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: selfdrive/controls/lib/events.py:148
msgid "MyPilot Unavailable"
msgstr "無法使用 MyPilot"

#: selfdrive/controls/lib/events.py:155 selfdrive/controls/lib/events.py:162
msgid "TAKE CONTROL IMMEDIATELY"
msgstr "即刻接管控制"

#: selfdrive/controls/lib/events.py:187 selfdrive/controls/lib/events.py:319
#: selfdrive/controls/lib/events.py:346 selfdrive/controls/lib/events.py:410
#: selfdrive/controls/lib/events.py:462 selfdrive/controls/lib/events.py:526
msgid "TAKE CONTROL"
msgstr "接管控制"

#: selfdrive/controls/lib/events.py:188
#, python-format
msgid "Steer Unavailable Below %s"
msgstr "橫向控制暫時失效，車速低於 %s"

#: selfdrive/controls/lib/events.py:196
#, python-format
msgid "Calibration in Progress: %(calPerc)d%%"
msgstr "正在校準中：%(calPerc)d%%"

#: selfdrive/controls/lib/events.py:197
#, python-format
msgid "Drive Above %s"
msgstr "車速請高於 %s"

#: selfdrive/controls/lib/events.py:204
msgid "Poor GPS reception"
msgstr "GPS 訊號不良"

#: selfdrive/controls/lib/events.py:205
msgid "If sky is visible, contact support"
msgstr "如果您不在地下室/隧道，請聯系客服"

#: selfdrive/controls/lib/events.py:205
msgid "Check GPS antenna placement"
msgstr "請檢查 GPS 天線位置"

#: selfdrive/controls/lib/events.py:210
msgid "Cruise Mode Disabled"
msgstr "巡航模式關閉"

#: selfdrive/controls/lib/events.py:212
msgid "Main Switch Off"
msgstr "主開關已關閉"

#: selfdrive/controls/lib/events.py:222
msgid "DEBUG ALERT"
msgstr "除錯用警示訊息"

#: selfdrive/controls/lib/events.py:224
msgid "Speed Below %s"
msgstr "車速低於 %s"

#: selfdrive/controls/lib/events.py:230
msgid "Be ready to take over at any time"
msgstr "請準備好隨時接管"

#: selfdrive/controls/lib/events.py:231 selfdrive/controls/lib/events.py:239
#: selfdrive/controls/lib/events.py:247 selfdrive/controls/lib/events.py:255
msgid "Always keep hands on wheel and eyes on road"
msgstr "將手放在方向盤上並持續監視路況"

#: selfdrive/controls/lib/events.py:238
msgid "WARNING: This branch is not tested"
msgstr "注意：這個分支未經過測試"

#: selfdrive/controls/lib/events.py:246
msgid "Dashcam mode"
msgstr "行車記錄模式"

#: selfdrive/controls/lib/events.py:254
msgid "Dashcam mode for unsupported car"
msgstr "行車記錄模式 (尚未支援車種)"

#: selfdrive/controls/lib/events.py:262 selfdrive/controls/lib/events.py:287
msgid "Dashcam Mode"
msgstr "行車記錄模式"

#: selfdrive/controls/lib/events.py:270
msgid "Stock LKAS is turned on"
msgstr "原廠 LKAS 已開啟"

#: selfdrive/controls/lib/events.py:271
msgid "Turn off stock LKAS to engage"
msgstr "需關閉原廠 LKAS 才能啟用"

#: selfdrive/controls/lib/events.py:279
msgid "Community Feature Detected"
msgstr "檢測到社群開發功能"

#: selfdrive/controls/lib/events.py:280
msgid "Enable Community Features in Developer Settings"
msgstr "請至開發人員設定裡啟用社群開發功能"

#: selfdrive/controls/lib/events.py:288
msgid "Car Unrecognized"
msgstr "無法辨識車款"

#: selfdrive/controls/lib/events.py:295 selfdrive/controls/lib/events.py:303
#: selfdrive/controls/lib/events.py:311
msgid "BRAKE!"
msgstr "剎車！"

#: selfdrive/controls/lib/events.py:296
msgid "Stock AEB: Risk of Collision"
msgstr "有碰撞的風險"

#: selfdrive/controls/lib/events.py:304
msgid "Stock FCW: Risk of Collision"
msgstr "有碰撞的風險"

#: selfdrive/controls/lib/events.py:312
msgid "Risk of Collision"
msgstr "有碰撞的風險"

#: selfdrive/controls/lib/events.py:320
msgid "Lane Departure Detected"
msgstr "偏離車道"

#: selfdrive/controls/lib/events.py:329
msgid "MyPilot will not brake while gas pressed"
msgstr "在您踩著油門的時候 MyPilot 將不會剎車"

#: selfdrive/controls/lib/events.py:338
msgid "Vehicle Parameter Identification Failed"
msgstr "車子參數識別失敗"

#: selfdrive/controls/lib/events.py:347 selfdrive/controls/lib/events.py:527
#: selfdrive/controls/lib/events.py:530
msgid "Steering Temporarily Unavailable"
msgstr "橫向控制暫時失效"

#: selfdrive/controls/lib/events.py:354
msgid "KEEP EYES ON ROAD: Driver Distracted"
msgstr "注意路況：駕駛分心"

#: selfdrive/controls/lib/events.py:362
msgid "KEEP EYES ON ROAD"
msgstr "注意路況"

#: selfdrive/controls/lib/events.py:363
msgid "Driver Appears Distracted"
msgstr "駕駛分心"

#: selfdrive/controls/lib/events.py:370 selfdrive/controls/lib/events.py:394
msgid "DISENGAGE IMMEDIATELY"
msgstr "立即解除"

#: selfdrive/controls/lib/events.py:371
msgid "Driver Was Distracted"
msgstr "駕駛分心"

#: selfdrive/controls/lib/events.py:378
msgid "TOUCH STEERING WHEEL: No Face Detected"
msgstr "請觸碰方向盤：未偵測到駕駛面容"

#: selfdrive/controls/lib/events.py:386
msgid "TOUCH STEERING WHEEL"
msgstr "請觸碰方向盤"

#: selfdrive/controls/lib/events.py:387
msgid "Driver Is Unresponsive"
msgstr "駕駛沒有反應"

#: selfdrive/controls/lib/events.py:395
msgid "Driver Was Unresponsive"
msgstr "駕駛沒有反應"

#: selfdrive/controls/lib/events.py:402
msgid "CHECK DRIVER FACE VISIBILITY"
msgstr "請檢查駕駛面部的可見度"

#: selfdrive/controls/lib/events.py:403
msgid "Driver Monitor Model Output Uncertain"
msgstr "駕駛監控模型判斷不明確"

#: selfdrive/controls/lib/events.py:411
msgid "Resume Driving Manually"
msgstr "請自行恢復駕駛"

#: selfdrive/controls/lib/events.py:418
msgid "STOPPED"
msgstr "已停止"

#: selfdrive/controls/lib/events.py:419
msgid "Press Resume to Move"
msgstr "請按 RES 繼續"

#: selfdrive/controls/lib/events.py:430
msgid "Steer Left to Start Lane Change"
msgstr "請往左打方向盤切換至左車道"

#: selfdrive/controls/lib/events.py:431 selfdrive/controls/lib/events.py:439
#: selfdrive/controls/lib/events.py:447 selfdrive/controls/lib/events.py:455
msgid "Monitor Other Vehicles"
msgstr "請注意其它車輛"

#: selfdrive/controls/lib/events.py:438
msgid "Steer Right to Start Lane Change"
msgstr "請往右打方向盤切換至右車道"

#: selfdrive/controls/lib/events.py:446
msgid "Car Detected in Blindspot"
msgstr "盲點偵測到車輛"

#: selfdrive/controls/lib/events.py:454
msgid "Changing Lane"
msgstr "切換車道中"

#: selfdrive/controls/lib/events.py:463
msgid "Turn Exceeds Steering Limit"
msgstr "彎道超過橫向操控限制"

#: selfdrive/controls/lib/events.py:469
msgid "Fan Malfunction"
msgstr "風扇故障"

#: selfdrive/controls/lib/events.py:469 selfdrive/controls/lib/events.py:473
#: selfdrive/controls/lib/events.py:477
msgid "Contact Support"
msgstr "請聯系客服"

#: selfdrive/controls/lib/events.py:473
msgid "Camera Malfunction"
msgstr "相機故障"

#: selfdrive/controls/lib/events.py:477
msgid "GPS Malfunction"
msgstr "GPS 故障"

#: selfdrive/controls/lib/events.py:500
msgid "Brake Hold Active"
msgstr "駐車煞車已啟用"

#: selfdrive/controls/lib/events.py:505
msgid "Park Brake Engaged"
msgstr "電子駐車已啟動"

#: selfdrive/controls/lib/events.py:510
msgid "Pedal Pressed During Attempt"
msgstr "啟用時偵測到駕駛踩踏油門/剎車"

#: selfdrive/controls/lib/events.py:511
msgid "Camera Frame Rate Low"
msgstr "相機幀率過低"

#: selfdrive/controls/lib/events.py:521
msgid "Enable Adaptive Cruise"
msgstr "啟用主動定速巡航"

#: selfdrive/controls/lib/events.py:536
msgid "Out of Storage"
msgstr "儲存不足"

#: selfdrive/controls/lib/events.py:540
msgid "Out of Storage Space"
msgstr "儲存空間不足"

#: selfdrive/controls/lib/events.py:545
msgid "Speed Too Low"
msgstr "車速過慢"

#: selfdrive/controls/lib/events.py:550 selfdrive/controls/lib/events.py:554
msgid "No Data from Device Sensors"
msgstr "未收到裝置傳感器數據"

#: selfdrive/controls/lib/events.py:551 selfdrive/controls/lib/events.py:562
#: selfdrive/controls/lib/events.py:650
msgid "Reboot your Device"
msgstr "請重啟裝置"

#: selfdrive/controls/lib/events.py:559
msgid "Release Pedal to Engage"
msgstr "釋放踏板"

#: selfdrive/controls/lib/events.py:562 selfdrive/controls/lib/events.py:563
msgid "Speaker not found"
msgstr "找不到音效裝置"

#: selfdrive/controls/lib/events.py:567
msgid "Distraction Level Too High"
msgstr "駕駛分心太多次"

#: selfdrive/controls/lib/events.py:572 selfdrive/controls/lib/events.py:576
msgid "System Overheated"
msgstr "系統過熱"

#: selfdrive/controls/lib/events.py:577
msgid "System overheated"
msgstr "系統過熱"

#: selfdrive/controls/lib/events.py:581 selfdrive/controls/lib/events.py:582
msgid "Gear not D"
msgstr "不在 D 檔位"

#: selfdrive/controls/lib/events.py:586
msgid "Calibration Invalid"
msgstr "校準無效"

#: selfdrive/controls/lib/events.py:586
msgid "Remount Device and Recalibrate"
msgstr "請將裝置放於新的位置並重新校準"

#: selfdrive/controls/lib/events.py:587
msgid "Calibration Invalid: Reposition Device and Recalibrate"
msgstr "校準無效：請將裝置放於新的位置並重新校準"

#: selfdrive/controls/lib/events.py:588
msgid "Calibration Invalid: Reposition Device & Recalibrate"
msgstr "校準無效：請將裝置放於新的位置並重新校準"

#: selfdrive/controls/lib/events.py:593 selfdrive/controls/lib/events.py:594
msgid "Calibration in Progress"
msgstr "正在校準中"

#: selfdrive/controls/lib/events.py:598
msgid "Door Open"
msgstr "車門未關"

#: selfdrive/controls/lib/events.py:599
msgid "Door open"
msgstr "車門未關"

#: selfdrive/controls/lib/events.py:603
msgid "Seatbelt Unlatched"
msgstr "安全帶未繫"

#: selfdrive/controls/lib/events.py:604
msgid "Seatbelt unlatched"
msgstr "安全帶未繫"

#: selfdrive/controls/lib/events.py:608 selfdrive/controls/lib/events.py:609
msgid "ESP Off"
msgstr "ESP 關閉"

#: selfdrive/controls/lib/events.py:613 selfdrive/controls/lib/events.py:614
msgid "Low Battery"
msgstr "電量過低"

#: selfdrive/controls/lib/events.py:618 selfdrive/controls/lib/events.py:619
msgid "Communication Issue between Processes"
msgstr "行程間出現通訊問題"

#: selfdrive/controls/lib/events.py:624
msgid "System Malfunction: Reboot Your Device"
msgstr "系統故障：請重啟您的裝置"

#: selfdrive/controls/lib/events.py:629 selfdrive/controls/lib/events.py:630
msgid "Radar Error: Restart the Car"
msgstr "雷達訊號錯誤：請重新發動車輛"

#: selfdrive/controls/lib/events.py:634 selfdrive/controls/lib/events.py:635
msgid "Driving model lagging"
msgstr "操控模型有延遲"

#: selfdrive/controls/lib/events.py:639 selfdrive/controls/lib/events.py:640
msgid "Model Output Uncertain"
msgstr "視覺模型判斷不明確"

#: selfdrive/controls/lib/events.py:644 selfdrive/controls/lib/events.py:645
msgid "Device Fell Off Mount"
msgstr "裝置掉落偵測"

#: selfdrive/controls/lib/events.py:649 selfdrive/controls/lib/events.py:651
msgid "Low Memory: Reboot Your Device"
msgstr "記憶體不足：請重啟您的裝置"

#: selfdrive/controls/lib/events.py:650
msgid "Low Memory"
msgstr "記憶體嚴重不足"

#: selfdrive/controls/lib/events.py:656 selfdrive/controls/lib/events.py:657
msgid "Controls Failed"
msgstr "控制發生錯誤"

#: selfdrive/controls/lib/events.py:661
msgid "Controls Mismatch"
msgstr "控制不匹配"

#: selfdrive/controls/lib/events.py:665 selfdrive/controls/lib/events.py:667
#: selfdrive/controls/lib/events.py:671
msgid "CAN Error: Check Connections"
msgstr "CAN 訊號錯誤：請檢查線路"

#: selfdrive/controls/lib/events.py:772
msgid "CAN Error"
msgstr "CAN 訊號錯誤"

#: selfdrive/controls/lib/events.py:782
msgid "CAN Bus Disconnected"
msgstr "CAN 訊號未連接"

#: selfdrive/controls/lib/events.py:784
msgid "CAN Bus Disconnected: Likely Faulty Cable"
msgstr "CAN 訊號未連接: 可能連接線束有故障"

#: selfdrive/controls/lib/events.py:788
msgid "CAN Bus Disconnected: Check Connections"
msgstr "CAN 訊號未連接: 請檢查"

#: selfdrive/controls/lib/events.py:675 selfdrive/controls/lib/events.py:681
msgid "LKAS Fault: Restart the Car"
msgstr "LKAS 錯誤：請重新發動車輛"

#: selfdrive/controls/lib/events.py:677
msgid "LKAS Fault: Restart the car to engage"
msgstr "LKAS 錯誤：請重新發動車輛"

#: selfdrive/controls/lib/events.py:685 selfdrive/controls/lib/events.py:691
#: selfdrive/controls/lib/events.py:755
msgid "Cruise Fault: Restart the Car"
msgstr "巡航系統錯誤：請重新發動車輛"

#: selfdrive/controls/lib/events.py:681
msgid "Low Communication Rate between Processes"
msgstr "進程之間消息通訊幀率過低"

#: selfdrive/controls/lib/events.py:687 selfdrive/controls/lib/events.py:751
msgid "Cruise Fault: Restart the car to engage"
msgstr "巡航系統錯誤：請重新發動車輛"

#: selfdrive/controls/lib/events.py:696
msgid ""
"Reverse\n"
"Gear"
msgstr "切換至倒車檔"

#: selfdrive/controls/lib/events.py:700 selfdrive/controls/lib/events.py:701
msgid "Reverse Gear"
msgstr "切換至倒車檔"

#: selfdrive/controls/lib/events.py:705
msgid "Cruise Is Off"
msgstr "巡航系統關閉"

#: selfdrive/controls/lib/events.py:709 selfdrive/controls/lib/events.py:710
msgid "Planner Solution Error"
msgstr "Planner Solution 錯誤"

#: selfdrive/controls/lib/events.py:714 selfdrive/controls/lib/events.py:715
#: selfdrive/controls/lib/events.py:716
msgid "Harness Malfunction"
msgstr "Harness 故障"

#: selfdrive/controls/lib/events.py:715
msgid "Check Hardware"
msgstr "請檢查硬體"

#: selfdrive/controls/lib/events.py:721 selfdrive/controls/lib/events.py:730
msgid "MyPilot Canceled"
msgstr "MyPilot 已取消"

#: selfdrive/controls/lib/events.py:722
msgid "No close lead car"
msgstr "前方沒有車輛"

#: selfdrive/controls/lib/events.py:725
msgid "No Close Lead Car"
msgstr "前方沒有車輛"

#: selfdrive/controls/lib/events.py:731
msgid "Speed too low"
msgstr "車速過慢"

#: selfdrive/controls/lib/events.py:738 selfdrive/controls/lib/events.py:743
msgid "Speed Too High"
msgstr "車速過快"

#: selfdrive/controls/lib/events.py:739
msgid "Model uncertain at this speed"
msgstr "當前車速下模型輸出不可靠"

#: selfdrive/controls/lib/events.py:744
msgid "Slow down to engage"
msgstr "請減速後再啟用"

#: selfdrive/controls/lib/events.py:760 selfdrive/controls/lib/events.py:764
#: selfdrive/controls/lib/events.py:765
msgid "System Overcooled"
msgstr "系統溫度過低"

#: selfdrive/controls/lib/events.py:771
msgid "WARNING"
msgstr "警告"

#: selfdrive/controls/lib/events.py:772
msgid "Grab wheel to start bypass"
msgstr "請減速後啟用"

#: selfdrive/controls/lib/events.py:779
msgid "BYPASSING"
msgstr "繞過時間限制中"

#: selfdrive/controls/lib/events.py:780
msgid "HOLD WHEEL"
msgstr "握好方向盤"

#: selfdrive/controls/lib/events.py:787
msgid "Bypassed!"
msgstr "時間限制已繞過！"

#: selfdrive/controls/lib/events.py:788
msgid "Release wheel when ready"
msgstr "準備好後請鬆開方向盤"

#~ msgid "WARNING: Original EON deprecated"
#~ msgstr "注意: 已停止支援壹加手機"

#~ msgid "Device will no longer update"
#~ msgstr "設備將不再升級"

#~ msgid "Slow down to resume operation"
#~ msgstr "請減速後再啟用"

#: selfdrive/controls/lib/events.py:236
msgid "Controls Initializing"
msgstr "正在初始化控制"

#: selfdrive/controls/lib/events.py:217
msgid "WARNING: No Exact Match on Car Model"
msgstr "警告：不完全匹配車型"

#: selfdrive/controls/lib/events.py:236
msgid "Closest Match:"
msgstr "最近匹配車型:"

#: selfdrive/controls/lib/events.py:236
msgid "Check All Connections"
msgstr "檢查所有連接"

#: selfdrive/controls/lib/events.py:236
msgid "MyPilot Not Available"
msgstr "MyPilot不可用"

#: selfdrive/controls/lib/events.py:236
msgid "Enable Community Features in Settings to Engage"
msgstr "在設置中打開社區功能開關"

#: selfdrive/controls/lib/events.py:236
msgid "Localizer unstable"
msgstr "航向儀不穩定"

#: selfdrive/controls/lib/events.py:236
msgid "Cruise Faulted"
msgstr "巡航故障"

#: selfdrive/controls/lib/events.py:236
msgid "Road Camera Error"
msgstr "車道攝像頭異常"

#: selfdrive/controls/lib/events.py:236
msgid "Driver Camera Error"
msgstr "駕駛員攝像頭異常"

#: selfdrive/controls/lib/events.py:236
msgid "Wide Road Camera Error"
msgstr "寬車道攝像頭異常"

#: selfdrive/controls/lib/events.py:236
msgid "USB Error: Reboot Your Device"
msgstr "USB異常: 請重啟您的設備"

#: selfdrive/controls/lib/events.py:842
msgid "LKAS Disabled: Enable LKAS to engage"
msgstr "LKAS已關閉: 請打開LKAS"

#: selfdrive/controls/lib/events.py:843
msgid "LKAS Disabled"
msgstr "LKAS已關閉"
